### PR TITLE
add pinv to jax.numpy frontend

### LINKED ISSUE #6207: pinv

### PR DESCRIPTION
**Pull request by minivet**
Close #6207

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by sharique1006** [comment]
@minivet, looks good to me!
